lib/biran/config_defaults.rb

```diff
@@ -11,11 +11,11 @@ def app_defaults_init
           shared_dir: configuration.shared_dir,
           base_dir: configuration.base_dir,
           use_capistrano: configuration.use_capistrano,
-          db_config: configuration.db_config,
-          secrets: configuration.secrets,
           bindings: configuration.bindings,
           vhost_public_dirname: configuration.vhost_public_dirname
-        }
+        },
+        db_config: configuration.db_config,
+        secrets: configuration.secrets,
```

Comment:
This is what actually fixes #60, in my opinion. These values should not have been under `app`

Reply:
If these values have to be defined you can add some validations. A sort of schema validation on the expected keys and values.

Reply:
They are defined already with defaults in the `config.rb` file. Using the `configuration` object pulls in those defined defaults or allows a user to set them via `Biran.configure`.   The trouble is that we were never using them in this location and I made the mistake of misreading this thinking `app_config_defaults[:secrets] was getting set because of this in here when really, it was `app_config_defaults[:app][:secrets]` that was getting set and then never being used.